src/hotspot/share/logging/logConfiguration.cpp

```diff
@@ -427,6 +427,7 @@ bool LogConfiguration::parse_log_arguments(const char* outputstr,
 
   ConfigurationLock cl;
   size_t idx;
+  bool is_added = false;
```

Comment:
Nit: `was_added` (or even just `added`)

Reply:
Thank you, David. I changed `is_added` to `added`.